test/cpp2rust/lib.cpp

```diff
@@ -1,7 +1,5 @@
 #include <iostream>
-#include <string_view>
 
 extern "C" void cpp_function(char const *name) {
-    std::string_view const name_sv = name;
-    std::cout << "Hello, " << name_sv << "! I'm C++!\n";
+    std::cout << "Hello, " << "my name is lib.cpp" << "! I'm C++!\n";

```

Comment:
```suggestion
    std::cout << "Hello, " << name << "! I'm C++!\n";
```

Reply:
Oops :-)
I'll change that. Sorry.

What do you prefer ? 
A 2nd `commit` or a `squash` ?